spec/factories/miq_dialog.rb

```diff
@@ -37,4 +37,14 @@
     name        "miq_provision_configured_system_foreman_dialogs"
     dialog_type "MiqProvisionConfiguredSystemWorkflow"
   end
+
+  factory :miq_dialog_aws_provision, :parent => :miq_dialog do
+    name        "miq_provision_amazon_dialogs_template"
+    dialog_type "MiqProvisionWorkflow"
+
+    content do
```

Comment:
But is it valid without the content? Could it be moved into the body of your tests?